packages/components/organisms/f-loyalty/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-loyalty",
   "description": "Fozzie Loyalty - provides a way for customers to collect loyalty stamps for restaurants",
-  "version": "0.6.0",
+  "version": "0.9.0",
```

Comment:
From 0.6.0 -> 0.9.0. Just curious for the reason behind the jump.

Reply:
I am lining up multiple components as separate PR's to make it easier to review, so I am just picking version numbers in the order in which I will merge the PR's. Hence the do not merge tag :)

Reply:
Is it worth converting the ones lined up for review, but not ready for merge as draft PR's?

(I've only just noticed, but you can convert PRs to a draft on the right of the PR, by reviewers where it says `Still in progress? Convert to draft`)